.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "4"
+  - "5"
+  - "6"
```

Comment:
Let's just do 4 and 6. We get next to nothing from testing 5 except longer build times.